### DISCUSSION TITLE
Can the plugin automatically unclaim player plots if the owner is inactive for 1 week

### DISCUSSION THREAD

**Discussion by RocketManKian** [GriefPrevention/GriefPrevention#967] [Q&A] [answered]
Hi there! I was just wondering; as seen in the wiki it is already setup for chest claims to automatically expire after 1 week if the owner is inactive. So... I was wondering if this is also possible to set this up for large land claims, so if the owner is inactive for a week their plot will be unclaimed. Thanks!

**Comment by bigpresh** [comment, marked as answer]
Yep, https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#expiring-inactive-claims

**Comment by bigpresh** [comment]
(A week is a bit harsh though!)

**Comment by RocketManKian** [comment]
Yeah haha just set it to 30 days. Just wanted to make sure as I thought this just expired them if they didn't expand, rather than weren't inactive :) Thanks
